pkgs/applications/version-management/gitea/default.nix

```diff
@@ -52,6 +53,12 @@ buildGoModule rec {
   outputs = [ "out" "data" ];
 
   postInstall = ''
+    # Create static gzip and brotli files
+    find public -type f -regextype posix-extended -iregex '.*\.(css|html|js|svg|ttf|txt)' | while read file; do
+      gzip -9 -n -c $file > $file.gz
+      brotli --best -f $file -o $file.br
+    done
```

Comment:
This can take a good 5 minutes to run and if the settings in nginx are disabled it is a complete waist of time. Can we move this into passthru and make the gitea nixos module pick it, if one of the recommneded settings is on?

/cc @Ma27

Reply:
Add optionn `withStatic`?

Reply:
I don't think this should be an option either because we don't need to recompile everything. It is an extra step that is good suited for passthru to extend the already built package but I didn't read the entire thread yet.